src/include/connection.h

```diff
@@ -285,12 +285,13 @@ struct __wt_connection_impl {
 	uint64_t  ckpt_time_recent;	/* Checkpoint time recent/total */
 	uint64_t  ckpt_time_total;
 
-#define	WT_CONN_STAT_ALL	0x01	/* "all" statistics configured */
-#define	WT_CONN_STAT_CLEAR	0x02	/* clear after gathering */
-#define	WT_CONN_STAT_FAST	0x04	/* "fast" statistics configured */
-#define	WT_CONN_STAT_JSON	0x08	/* output JSON format */
-#define	WT_CONN_STAT_ON_CLOSE	0x10	/* output statistics on close */
-#define	WT_CONN_STAT_SIZE	0x20	/* "size" statistics configured */
+#define	WT_CONN_STAT_ALL	0x01	/* All statistics configured */
+#define	WT_CONN_STAT_CACHE_WALK	0x02	/* Cache walk statistics configured */
+#define	WT_CONN_STAT_CLEAR	0x04	/* Clear statistics after gathering */
+#define	WT_CONN_STAT_FAST	0x08	/* Fast statistics configured */
+#define	WT_CONN_STAT_JSON	0x10	/* Output in JSON format */
+#define	WT_CONN_STAT_ON_CLOSE	0x20	/* Output statistics on close */
+#define	WT_CONN_STAT_SIZE	0x40	/* Size statistics configured */
```

Comment:
It surprised me on revisiting this code that these flags are also used in `WT_CURSOR_STAT`.  They are also a mix of statistics categories and behavior of the statistics logging thread.

How would you feel about renaming the category flags `WT_STAT_TYPE_*` and moving them into `flags.py`?